docs/requirements.txt

```diff
@@ -1,3 +1,4 @@
+docutils==0.15.2
```

Comment:
would prefer a comment explaining why we need to pin this (or just indicating that it is a transitive dep that we want to pin)

Reply:
👍  added a comment